exercises/concept/party-robot/.docs/introduction.md

```diff
@@ -1,5 +1,7 @@
 # Introduction
 
+Go programs are organized into packages. A package is a collection of source files in the same directory that are compiled together. Functions, types, variables, and constants defined in one source file are visible to all other source files within the same package. Similar to program languages like Java that use access modifiers such as `public` and `private`, Go determines if an item is exported and unexported through how it is declared. To allow an item to be called by code in other packages, the name must start with a capital letter.
```

Comment:
Can you split this into one line per sentence. A lot of the text isn't like this at the moment, but we're trying to change it as we touch things as this makes diffs much easier to read.

Also this is the exercise's introduction.md - at least some of this content needs adding to the concept's introduction.md document (`concepts/packages/introduction.md`) and also the about.md document (`concepts/packages/about.md`).

Reply:
@bobtfish Thanks for the helpful comments. 

I'm new to the `concepts` feature - how do I know what information to put in the `about.md` vs `introduction.md` ?

Reply:
Hey @btsui3 - there's some guidance here: https://exercism.org/docs/building/tracks/concepts

tl;dr: `introduction.md` should only contain material needed to solve the related exercise, no links - this is what's shown before the student has completed the exercise. `about.md` can contain a superset of this information / a broader picture and will be shown once the exercise is completed, it can / should link out to other documentation.

They are often extremely similar at the moment as we've been beating things into shape in Go, over time they'll probably diverge more!